test/e2e/page-objects/pages/account-list-page.ts

```diff
@@ -234,6 +247,72 @@ class AccountListPage {
     }
   }
 
+  async addAccount(accountType: ACCOUNT_TYPE, accountName: string) {
+    await this.driver.clickElement(this.createAccountButton);
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let addAccountButton: any;
+    switch (accountType) {
+      case ACCOUNT_TYPE.ETHEREUM:
+        addAccountButton = this.addEthereumAccountButton;
+        // await this.driver.clickElement(this.addEthereumAccountButton);
+
+        break;
+      case ACCOUNT_TYPE.BITCOIN:
+        addAccountButton = this.addBtcAccountButton;
+        // await this.driver.clickElement(this.addBtcAccountButton);
+        break;
+      case ACCOUNT_TYPE.SOLANA:
+        addAccountButton = this.addSolanaAccountButton;
+        // await this.driver.clickElement(this.addSolanaAccountButton);
+        break;
+      default:
+        console.log('Account type does not match with any of the options');
+        return;
+    }
+
+    await this.driver.clickElement(addAccountButton);
+    if (accountName) {
+      await this.driver.fill(this.accountNameInput, accountName);
+    }
+
+    await this.driver.clickElementAndWaitToDisappear(
+      this.addAccountConfirmButton,
+      5000,
+    );
+  }
+
+  async addNewSolanaAccount({
+    solanaAccountCreationEnabled = true,
+    accountName = '',
+  }: {
+    solanaAccountCreationEnabled?: boolean;
+    accountName?: string;
+  } = {}): Promise<void> {
+    console.log(
+      `Adding new Solana account${
+        accountName ? ` with custom name: ${accountName}` : ' with default name'
+      }`,
+    );
+    if (solanaAccountCreationEnabled) {
+      await this.driver.clickElement(this.addSolanaAccountButton);
+      // needed to mitigate a race condition with the state update
+      // there is no condition we can wait for in the UI
+      if (accountName) {
+        await this.driver.fill(this.accountNameInput, accountName);
+      }
+      await this.driver.clickElementAndWaitToDisappear(
+        this.addAccountConfirmButton,
+        5000,
+      );
+    } else {
+      const createButton = await this.driver.findElement(
+        this.addSolanaAccountButton,
+      );
+      assert.equal(await createButton.isEnabled(), false);
+      await this.driver.clickElement(this.closeAccountModalButton);
+    }
+  }
+
```

Comment:
I believe this is dead code now

```suggestion
```

Reply:
We should probably remove `addNewBtcAccount` now that we are using the common helper